FilecoinFoundationWeb/filecoin-foundation
# [UXIT-1638] Allocators Sort

## 📝 Description

This PR adds a sorting feature to the allocators' table.

## 🛠️ Key Changes

- Introduced `SelectSort` to interact with the column's sorting state
- Added a local `constants/sorting` file to manage the sort defaults, and highlight the dependency between the table state and the component's default sort option.

## 🧪 How to Test

- Go to `/filecoin-plus/allocators` and try sorting the table
- Change the default table sort in `constants/sorting` and make sure nothing breaks. Switching `desc: false` to `true` should automatically update the component's default sort option.

## 📸 Screenshots

![CleanShot 2024-10-29 at 13 39 17@2x](https://github.com/user-attachments/assets/24d2a2c2-99b7-4423-bf7f-a99f3a86a839)

## 🔖 Resources

- About [satisfies](https://www.youtube.com/watch?v=r1L35zxZQPE)
